index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Colin's Website</title>
+	</head>
+	<body>
+		<nav>
+			<a class="navbar" href="#info">About Me</a>
+			<a class="navbar" href="#portfolio">Portfolio</a>
+			<a class="navbar" href="#contact">Contact</a>
```

Comment:
the class name "navbar" tells me that this element is a navbar (so here we've got 3 navbars). whereas it is not, it is a link within a navbar. so it needs a more semantic name